packages/provider/src/language-model/v2/language-model-v2-file.ts

```diff
@@ -22,4 +23,9 @@ as base64 encoded strings. If the API returns binary data, the file data should
 be returned as binary data.
  */
   data: string | Uint8Array;
+
+  /**
+   * Additional provider-specific metadata for the file.
+   */
+  providerMetadata?: SharedV2ProviderMetadata;
```

Comment:
This is a proposal -- if this is non-breaking and feasible, we need to add documentation for the type change to this PR. Waiting for feedback before doing so, since I realize we may want to revert this and consider as work for the future instead.